### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Queues_QueueTable_Screenreader : The table header content is announced multiple times for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/AzureStorageExplorer#3509]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yMAS;#A11YTCS;#Azure Tools Storage Explorer;#WCAG1.3.1;#MAC;#A11ySev2;#Benchmark;#Screenreader;#VoiceOver;#FTP;#BM-Azure Tools Storage Explorer-MAC-Aug2020;

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version: 1.15.0
AzCopy Version: 10.6.0
OS Version: macOS Catalina Version 10.15.6
Screen Reader: Voiceover

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Queues" of the selected subscription and expand it.
5. Navigate to any queue and click on it.
6. Start screen reader and navigate to table headers then observe if it announced two or three times or not.

### Actual:
When screen reader users navigate to the table the table header content is announced for two or three times which is very confusing for them while navigation.

This issue is observed in the below scenarios:
1. Blob Container Table - Table
2. FileShares - Filesharetable
3. Disk - DiskInfoTable

### Expected:
The table header content should be announced just once for screen reader users while they navigate through the table.

### User Impact:
If the screen reader users are announced with incorrect data for the content on the screen they will get confused with the information that is conveyed to them as compare to the information that is visually available on the screen.

### Attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-manai_microsoft_com/EepnJXkOeS5KuFsb_hFAOMMBYm3yeAXeAAzxYKz0wOCL7g?e=9D7COZ

**Comment by craxal** [comment]
@MayuriNaik I am having trouble reproducing. My screen reader says very different things than what yours does. Can you please elaborate on:
- The settings you have configured for your screen reader.
- The key combinations you are using to navigate.

![image](https://user-images.githubusercontent.com/3473026/92822530-1d87ef00-f381-11ea-8bd2-1bd4c83666f3.png)

**Comment by MayuriNaik** [comment]
Closing this bug as it is discussed that it’s a regular occurrence for screen readers to read headers two times. It is likely to screen readers are reading the ARIA label and the content, which is why we get multiple readings.

**Comment by Raghu-addepalli** [comment]
GitHubTags : #Closed;
